Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyBoolComboBoxCtrl.cpp

```diff
@@ -13,73 +13,38 @@ AZ_PUSH_DISABLE_WARNING(4251, "-Wunknown-warning-option") // 4251: 'QLayoutItem:
 #include <QtWidgets/QHBoxLayout>
 AZ_POP_DISABLE_WARNING
 
-//just a test to see how it would work to pop a dialog
+#pragma optimize("", off)
```

Comment:
Same here.. lots of pragma's that seem like you probably didn't mean to check them in.